hooks/pre/30-upgrade.rb

```diff
@@ -1,3 +1,13 @@
+def stop_services
+  Kafo::Helpers.execute('katello-service stop')
+end
+
+def start_mongo
+  status = Kafo::Helpers.execute('service-wait mongod status')
+  Kafo::Helpers.execute('service-wait mongod start') unless status
```

Comment:
I've been looking at making katello-service a little more intelligent. maybe one of the things I can add is an "--exclude" option so we can skip mongo in the first step in stopping everything else.